nix/automation/pipelines.nix

```diff
@@ -131,9 +141,33 @@
               '''
           '';
         }
+        {
+          destination = "/local/aarch64/default.nix";
+          data = ''
+            let
+              nixpkgs = __getFlake github:NixOS/nixpkgs/6107f97012a0c134c5848125b5aa1b149b76d2c9;
+              pkgs = nixpkgs.legacyPackages.aarch64-darwin;
+            in
+              pkgs.runCommand "foo" {} '''
+                {
+                  echo '{{timestamp}}' # to force a new build
+                  uname
+                } > $out
+              '''
+          '';
+        }
       ];
 
-      driver = "exec";
+      driver = config.actionRun.facts.trigger.value."ci-world/test-darwin-nix-remote-builders";
+
+      # For when host testing isolation is required:
+      # constraints = [
+      #   {
+      #     attribute = "\${attr.unique.network.ip-address}";
+      #     operator = "=";
+      #     value = "a.b.c.d";
+      #   }
+      # ];
```

Comment:
This could also be taken from the input fact so code has to be changed to use it. Not a blocker though.